--- cmor-2.9.1/Makefile.in	2014-06-24 23:19:00.000000000 +1200
+++ cmor-2.9.1-new/Makefile.in	2019-02-07 14:22:40.953128530 +1300
@@ -153,8 +153,8 @@
 	@env TEST_NAME=Test/test_python_joerg_8.py make test_a_python
 #	@env TEST_NAME=Test/test_python_joerg_9.py make test_a_python
 	@env TEST_NAME=Test/test_python_joerg_10.py make test_a_python
-        @env TEST_NAME=Test/test_python_joerg_11.py make test_a_python
-        @env TEST_NAME=Test/test_python_joerg_12.py make test_a_python
+	@env TEST_NAME=Test/test_python_joerg_11.py make test_a_python
+	@env TEST_NAME=Test/test_python_joerg_12.py make test_a_python
 	@env TEST_NAME=Test/test_python_YYYMMDDHH_exp_fmt.py make test_a_python
 	@env TEST_NAME=Test/test_python_region.py make test_a_python
 	@env TEST_NAME=Test/jamie_hybrid_height.py make test_a_python
